tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/SmokeTests/SandboxAutomaticInstrumentationSmokeTest.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="SandboxAutomaticInstrumentationSmokeTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NET461
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.SmokeTests
+{
+    public class SandboxAutomaticInstrumentationSmokeTest : SmokeTestBase
+    {
+        public SandboxAutomaticInstrumentationSmokeTest(ITestOutputHelper output)
+            : base(output, "Sandbox.AutomaticInstrumentation")
+        {
+        }
+
+        [Fact(Skip = "This test fails because the startup hook throws a System.Security.VerificationException")]
+        [Trait("Category", "Smoke")]
+        public void NoExceptions()
```

Comment:
Don't understand why we'd introduce a new test just to skip it.

Reply:
Yeah that's a good point, this is a new scenario that I didn't want to forget about, but it will definitely fail. Maybe I should assert that it fails instead of skipping it?

Reply:
Added commit de76f2e98a080230989298f08f7caaefcba04068 to run the test in CI and validate that it fails. This means if we ever get to fixing it, we'll have to update the test to expect success